Description: Change name of shared library
Author: Anton Gladky <gladky.anton@gmail.com>
Last-Update: 2012-06-20

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1027,7 +1027,11 @@
 # want to build the lib we need to recompile everything (the first
 # time--it's a separate target)
 add_library(lib STATIC EXCLUDE_FROM_ALL ${GMSH_SRC})
-set_target_properties(lib PROPERTIES OUTPUT_NAME Gmsh)
+set_target_properties(lib PROPERTIES 
+  OUTPUT_NAME gmsh
+  SOVERSION "${GMSH_MAJOR_VERSION}"
+  )
+
 if(MSVC)
   set_target_properties(lib PROPERTIES DEBUG_POSTFIX d) 
   if(ENABLE_MSVC_STATIC_RUNTIME)
@@ -1037,7 +1041,11 @@
 
 # shared library target: we don't build it by default either
 add_library(shared SHARED EXCLUDE_FROM_ALL ${GMSH_SRC})
-set_target_properties(shared PROPERTIES OUTPUT_NAME Gmsh)
+set_target_properties(shared PROPERTIES 
+  OUTPUT_NAME gmsh
+  SOVERSION "${GMSH_MAJOR_VERSION}"
+  )
+
 if(HAVE_LAPACK AND LAPACK_FLAGS)
   set_target_properties(shared PROPERTIES LINK_FLAGS ${LAPACK_FLAGS})
 endif(HAVE_LAPACK AND LAPACK_FLAGS)
