Description: delete gl2ps and use packaged version.
Author: Anton Gladky <gladky.anton@gmail.com>, Julian Taylor <jtaylor.debian@googlemail.com>
Last-Update: 2012-11-28

Index: gmsh-2.8.5-source/Graphics/CMakeLists.txt
===================================================================
--- gmsh-2.8.5-source.orig/Graphics/CMakeLists.txt
+++ gmsh-2.8.5-source/Graphics/CMakeLists.txt
@@ -15,7 +15,6 @@ set(SRC
   drawScales.cpp 
   drawGraph2d.cpp 
   drawGlyph.cpp 
-  gl2ps.cpp
   gl2gif.cpp
   gl2jpeg.cpp
   gl2png.cpp
Index: gmsh-2.8.5-source/CMakeLists.txt
===================================================================
--- gmsh-2.8.5-source.orig/CMakeLists.txt
+++ gmsh-2.8.5-source/CMakeLists.txt
@@ -1360,7 +1360,7 @@ else(HAVE_FLTK)
     add_executable(gmsh Common/Main.cpp ${GMSH_SRC})
   endif(ENABLE_BUILD_DYNAMIC)
 endif(HAVE_FLTK)
-target_link_libraries(gmsh ${LINK_LIBRARIES})
+target_link_libraries(gmsh ${LINK_LIBRARIES} -lgl2ps -lGL -lglut)
 
 # Windows specific linker options
 if(WIN32 AND NOT MSVC OR CYGWIN)
