Description: delete gl2ps and use packaged version.
Author: Anton Gladky <gladky.anton@gmail.com>, Julian Taylor <jtaylor.debian@googlemail.com>
Last-Update: 2011-12-09

--- a/Graphics/CMakeLists.txt
+++ b/Graphics/CMakeLists.txt
@@ -15,7 +15,6 @@
   drawScales.cpp 
   drawGraph2d.cpp 
   drawGlyph.cpp 
-  gl2ps.cpp
   gl2gif.cpp
   gl2jpeg.cpp
   gl2png.cpp
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -933,7 +933,7 @@
   message("WARNING: By enabling ENABLE_MSVC_STATIC_RUNTIME, shared library wont link. "
           "Change in msvc /MT flag to /MD in the shared project properties")
 endif(MSVC AND ENABLE_MSVC_STATIC_RUNTIME)		  
-target_link_libraries(shared ${LINK_LIBRARIES})
+target_link_libraries(shared ${LINK_LIBRARIES} -lgl2ps -lGL -lglut)
 
 # binary targets
 if(HAVE_FLTK)
@@ -948,7 +948,7 @@
   add_executable(gmsh_dynamic EXCLUDE_FROM_ALL Common/Main.cpp)
   target_link_libraries(gmsh_dynamic shared)
 endif(HAVE_FLTK)
-target_link_libraries(gmsh ${LINK_LIBRARIES})
+target_link_libraries(gmsh ${LINK_LIBRARIES} -lgl2ps -lGL -lglut)
 
 # increase stack to 16Mb on Windows to avoid overflows in recursive
 # tet classification for large 3D Delaunay grids + force static
