Description: Enable TetGen
Author: Anton Gladky <gladk@debian.org>
Last-Update: 2014-02-10

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -760,6 +760,7 @@
       list(APPEND EXTERNAL_LIBRARIES ${TETGEN_LIB})
       list(APPEND EXTERNAL_INCLUDES ${TETGEN_INC})
       set_config_option(HAVE_TETGEN "Tetgen")
+      add_definitions(-DTETLIBRARY)
     endif(TETGEN_LIB AND TETGEN_INC)
   endif(ENABLE_TETGEN AND EXISTS ${CMAKE_CURRENT_SOURCE_DIR}/contrib/Tetgen1.5/tetgen.h)
   if(HAVE_TETGEN)
@@ -1314,7 +1315,7 @@
     message(STATUS "Note: By enabling ENABLE_MSVC_STATIC_RUNTIME, shared library "
             "won't link. In MSVC change /MT to /MD in the shared project properties")
   endif(MSVC AND ENABLE_MSVC_STATIC_RUNTIME)		  
-  target_link_libraries(shared ${LINK_LIBRARIES} -shared -lgl2ps -lGL -lglut)
+  target_link_libraries(shared ${LINK_LIBRARIES} -shared -lgl2ps -lGL -lglut -ltet)
 endif(ENABLE_BUILD_SHARED OR ENABLE_BUILD_DYNAMIC OR 
       ENABLE_WRAP_PYTHON OR ENABLE_WRAP_JAVA)
 
@@ -1337,7 +1338,7 @@
     add_executable(gmsh Common/Main.cpp ${GMSH_SRC})
   endif(ENABLE_BUILD_DYNAMIC)
 endif(HAVE_FLTK)
-target_link_libraries(gmsh ${LINK_LIBRARIES} -lgl2ps -lGL -lglut)
+target_link_libraries(gmsh ${LINK_LIBRARIES} -lgl2ps -lGL -lglut -ltet)
 
 # Windows specific linker options
 if(WIN32 AND NOT MSVC OR CYGWIN)
