Description: Fix FTBFS against gcc-4.7
Author: Anton Gladky <gladky.anton@gmail.com>
Last-Update: 2012-05-24

--- a/contrib/bamg/bamglib/Mesh2.cpp
+++ b/contrib/bamg/bamglib/Mesh2.cpp
@@ -3201,7 +3201,7 @@
   if (inbvx) {
     vertices=new Vertex[nbvx];
     assert(vertices);
-    ordre=new (Vertex* [nbvx]);
+    ordre=new (Vertex* [this->nbvx]);
     assert(ordre);
     triangles=new Triangle[nbtx];
     assert(triangles);}
@@ -3977,7 +3977,7 @@
 Edge** Triangles::MakeGeometricalEdgeToEdge()
  {
   assert(Gh.nbe);
-  Edge **e= new (Edge* [Gh.nbe]);
+  Edge **e= new (Edge* [this->Gh.nbe]);
   
   Int4 i;
   for ( i=0;i<Gh.nbe ; i++)
--- a/contrib/bamg/bamglib/MeshRead.cpp
+++ b/contrib/bamg/bamglib/MeshRead.cpp
@@ -119,7 +119,7 @@
 	   nbvx=nbv;
 	   vertices=new Vertex[nbvx];
 	   assert(vertices);
-	   ordre=new (Vertex* [nbvx]);
+	   ordre=new (Vertex* [this->nbvx]);
 	   assert(ordre);
 	   
 	   nbiv = nbv;
@@ -399,7 +399,7 @@
      triangles =new Triangle[nbtx];
      assert(triangles);
      vertices=new Vertex[nbvx];
-     ordre=new (Vertex* [nbvx]);
+     ordre=new (Vertex* [this->nbvx]);
      
      for (     i=0;i<nbt;i++)
        {
@@ -448,7 +448,7 @@
   triangles =new Triangle[nbtx];
   assert(triangles);
   vertices=new Vertex[nbvx];
-  ordre=new (Vertex* [nbvx]);
+  ordre=new (Vertex* [this->nbvx]);
   
 
   Int4 i;
@@ -547,7 +547,7 @@
   triangles =new Triangle[nbtx];
   assert(triangles);
   vertices=new Vertex[nbvx];
-  ordre=new (Vertex* [nbvx]);
+  ordre=new (Vertex* [this->nbvx]);
 
 
  f_in >> l;
@@ -711,7 +711,7 @@
   triangles =new Triangle[nbtx];
   assert(triangles);
   vertices=new Vertex[nbvx];
-  ordre=new (Vertex* [nbvx]);
+  ordre=new (Vertex* [this->nbvx]);
   Int4 k=0;
   
   for ( i=0;i<ne;i++) 
@@ -774,7 +774,7 @@
      triangles =new Triangle[nbtx];
      assert(triangles);
      vertices=new Vertex[nbvx];
-     ordre=new (Vertex* [nbvx]);
+     ordre=new (Vertex* [this->nbvx]);
       edges = new Edge[nbe];
      for ( i=0;i<nbv;i++)
 	{
@@ -822,7 +822,7 @@
      triangles =new Triangle[nbtx];
      assert(triangles);
      vertices=new Vertex[nbvx];
-     ordre=new (Vertex* [nbvx]);
+     ordre=new (Vertex* [this->nbvx]);
      Int4 j;
       for ( i=0;i<nbv;i++)
 	{
