Desctiption: fix compilation of api_demos
Author: Christophe Trophime <christophe.trophime@lncmi.cnrs.fr>
Reviewed-by: Anton Gladky <gladky.anton@gmail.com>
Last-Update: 2012-03-11

--- gmsh-2.5.1~beta2~svn12143~dfsg.orig/Geo/STensor3.h
+++ gmsh-2.5.1~beta2~svn12143~dfsg/Geo/STensor3.h
@@ -334,7 +334,7 @@ inline SVector3 operator* (const STensor
     for (int j=0; j<3; j++)
       temp[i]+= t(i,j)*v[j];
   return temp;
-};
+}
 
 inline SVector3 operator* (const SVector3& v, const STensor3& t){
   SVector3 temp(0.,0.,0.);
@@ -342,7 +342,7 @@ inline SVector3 operator* (const SVector
     for (int j=0; j<3; j++)
       temp[j]+= v[i]*t(i,j);
   return temp;
-};
+}
 
 
 inline STensor3 operator*(const STensor3 &t, double m)
