Desctiption: Removes pedantic warnings
Author: Christophe Trophime <christophe.trophime@lncmi.cnrs.fr>
Reviewed-by: Anton Gladky <gladky.anton@gmail.com>
Last-Update: 2012-03-11

--- a/Geo/STensor3.h
+++ b/Geo/STensor3.h
@@ -332,7 +332,7 @@
     for (int j=0; j<3; j++)
       temp[i]+= t(i,j)*v[j];
   return temp;
-};
+}
 
 inline SVector3 operator* (const SVector3& v, const STensor3& t){
   SVector3 temp(0.,0.,0.);
@@ -340,7 +340,7 @@
     for (int j=0; j<3; j++)
       temp[j]+= v[i]*t(i,j);
   return temp;
-};
+}
 
 
 inline STensor3 operator*(const STensor3 &t, double m)
