Index: gmsh-2.1.1/contrib/Netgen/libsrc/general/mystring.hpp
===================================================================
--- gmsh-2.1.1.orig/contrib/Netgen/libsrc/general/mystring.hpp	2005-09-21 19:29:38.000000000 +0200
+++ gmsh-2.1.1/contrib/Netgen/libsrc/general/mystring.hpp	2008-04-10 06:53:11.000000000 +0200
@@ -19,6 +19,7 @@
 #ifndef MYSTRING__H
 #define MYSTRING__H
 
+
 class Point3d;
 class Vec3d;
 
Index: gmsh-2.1.1/contrib/Netgen/libsrc/general/symbolta.hpp
===================================================================
--- gmsh-2.1.1.orig/contrib/Netgen/libsrc/general/symbolta.hpp	2008-02-22 07:49:34.000000000 +0100
+++ gmsh-2.1.1/contrib/Netgen/libsrc/general/symbolta.hpp	2008-04-10 06:53:11.000000000 +0200
@@ -1,6 +1,7 @@
 #ifndef FILE_SYMBOLTA
 #define FILE_SYMBOLTA
 
+#include <string.h>
 
 /**************************************************************************/
 /* File:   symbolta.hh                                                    */
Index: gmsh-2.1.1/contrib/Netgen/libsrc/include/mystdlib.h
===================================================================
--- gmsh-2.1.1.orig/contrib/Netgen/libsrc/include/mystdlib.h	2006-06-08 14:32:08.000000000 +0200
+++ gmsh-2.1.1/contrib/Netgen/libsrc/include/mystdlib.h	2008-04-10 06:53:11.000000000 +0200
@@ -1,6 +1,7 @@
 #ifndef FILE_MYSTDLIB
 #define FILE_MYSTDLIB
 
+#include <cstring>
 
 #include <iostream>
 #include <iomanip>
Index: gmsh-2.1.1/contrib/Netgen/libsrc/opti/bfgs.cpp
===================================================================
--- gmsh-2.1.1.orig/contrib/Netgen/libsrc/opti/bfgs.cpp	2005-09-21 19:29:38.000000000 +0200
+++ gmsh-2.1.1/contrib/Netgen/libsrc/opti/bfgs.cpp	2008-04-10 06:53:11.000000000 +0200
@@ -8,6 +8,7 @@
 /*                                                                         */
 /***************************************************************************/
 
+#include <cstring>
 #include <mystdlib.h>
 #include <myadt.hpp> 
 
Index: gmsh-2.1.1/contrib/Netgen/libsrc/opti/linopt.cpp
===================================================================
--- gmsh-2.1.1.orig/contrib/Netgen/libsrc/opti/linopt.cpp	2005-09-21 19:29:38.000000000 +0200
+++ gmsh-2.1.1/contrib/Netgen/libsrc/opti/linopt.cpp	2008-04-10 06:53:11.000000000 +0200
@@ -1,3 +1,4 @@
+#include <cstring>
 #include <mystdlib.h>
 #include <myadt.hpp>
 
Index: gmsh-2.1.1/Fltk/Message.cpp
===================================================================
--- gmsh-2.1.1.orig/Fltk/Message.cpp	2008-04-10 06:53:39.000000000 +0200
+++ gmsh-2.1.1/Fltk/Message.cpp	2008-04-10 06:53:57.000000000 +0200
@@ -19,7 +19,8 @@
 // 
 // Please report all bugs and problems to <gmsh@geuz.org>.
 
-#include <string.h>
+#include <cstring>
+#include <cstdlib>
 #include <signal.h>
 #include "Message.h"
 #include "GmshUI.h"
